Packs/Zimperium/Integrations/Zimperium/Zimperium_description.md

```diff
@@ -0,0 +1,2 @@
+Zimperium uses the rsql to query for events, devices and users.
```

Comment:
this is the description to this product?

Reply:
No, this is a detailed description if a customer wants to configure the fetch query or the queries for the search commands.